Parser/pegen/pegen.c

```diff
@@ -2058,7 +2058,7 @@ _PyPegen_make_module(Parser *p, asdl_seq *a) {
 // Error reporting helpers
 
 expr_ty
-_PyPegen_get_invalid_target(expr_ty e)
+_PyPegen_get_invalid_target(expr_ty e, int del_targets)
```

Comment:
Can you please adopt a naming convention that makes it clear that this is effectively a bool? I was a bit baffled by this for a while. A venerable convention is to use names like `is_foo` or `has_bar`.

Reply:
Not sure, if `is_del` is okay, because the questions "What's del, the target, which target?" comes to mind. What do you think?